bedrock/mozorg/templates/mozorg/internet-health.html

```diff
@@ -81,6 +81,17 @@
     <section id="innovation" class="even">
       <div class="content">
         <div class="inner-container">
+        {% if l10n_has_tag('internet_health_digital_inclusion') %}
```

Comment:
Note for @peiying2: Ignore this for now, we already have this text in the file (I guess wrong copy and paste)